code/modules/mob/living/simple_animal/hostile/ordeal/_ordeal.dm

```diff
@@ -8,6 +8,7 @@
 	aggro_vision_range = 20
 	var/datum/ordeal/ordeal_reference
 	var/ordeal_remove_ondeath = TRUE
+	var/dawn = FALSE
```

Comment:
Definitely not the best way to do this. I'll go into further detail in another comment.

Reply:
So I forgot to go into further detail, more or less instead of the spears doing a check for a new variable being manually glued to every ordeal mob you could either:
1. Just do a type-list based check, where the spear gets bonuses against certain types of mobs
or
2. Create an actual threat-level system for ordeals

Reply:
I think they are now fixed!